test/language-transformer-cycles.test.js

```diff
@@ -168,5 +168,5 @@ describe.each(languagesWithTransforms)('Cycles Test $iso', ({languageTransforms}
                 deinflectionNodes.push(newDeinflectionNode);
             }
         }
-    });
+    }, {timeout: 60 * 1000});
```

Comment:
Is there a particular reason for the timeout and 60 seconds?

Reply:
Upgrading vitest from 2 -> 3 introduced a default test timeout of 5s. I think 30s would work as well.